content/en/graphing/metrics/distributions.md

```diff
@@ -37,67 +37,23 @@ After electing to apply percentile aggregations on a distribution metric, these
-
-Additionally, this metric has been tagged with `Status`, which has 2 values: `Status:Success` or `Status:Fail`, and `Result`, which also has 2 values: `Result:Open` or `Result:Close`.
-
-##### Scenario 1
-
-By default, Datadog creates aggregations for `my.service.latency` for each combination of custom metric tags `Status` and `Result`.
-
-You could then query, for example `{my.service.latency for Status:success, Result:closed}`, but not `{my.service.latency for Availability-Zone: US-East-1a}`
-
-This creates (2+1) * (2+1) * 10 = 90 timeseries.
-
-##### Scenario 2
-
-You aggregate on `{Status, Result, Host}` instead of the defaults.
-Available queries include, eg,:
-
-* `{my.service.latency for Status:success, Result:closed, Host: i-deadbeef}`
-* `{my.service.latency for Status:fail, host: i-deadbeef}`
+Note: The exclusion of tags is not supported in this whitelist-based customization of tags. Adding tags starting with `!` will not be accepted.
 
-The following is unavailable:
-
-* `{my.service.latency for Availability-Zone: US-East-1a}`
-
-This creates (2+1) * (2+1) * (500+1) * 10 = 45,090 timeseries.
-
-##### Scenario 3
-
-You aggregate by `{Availability-Zone, Status}`, in addition to `{Status, Result, Host}`.
-
-Available queries include:
-
-* `{my.service.latency for AZ: US-East-1a}`
-* `{my.service.latency for AZ: US-West-2b, Status: Success}`
-*  any in the previous scenario.
-
-The aggregate timeseries for `Availability-Zone` and `Host` has not been requested, so `{my.service.latency for AZ:US-East-1a, Host:i-deadbeef}` is unavailable.
-
-This creates (3 + 1) * (2 + 1) * 10 = 120, in addition to the 45,090, above, for a total of 45,210 timeseries.
-
-As you can see, aggregating by multiple sets of tags increases the number of timeseries additively, limiting timeseries growth.
-
-The following gif demonstrates inspecting the aggregations created for a metric, `intake.batch_api_keys_for_org`. You would query `{availability-zone, role}` or `{instance-type, role}`, but not `{availability-zone, instance-type}`.
+{{< img src="graphing/metrics/distributions/distribution_metric.png" alt="Distribution metric" responsive="true" style="width:80%;">}}
 
-{{< img src="graphing/metrics/distributions/Distros_Tagging.gif" alt="Distros_Tagging" responsive="true" style="width:80%;">}}
+## Counting Distribution Metrics
+Distribution metrics with percentile aggregations (p50,p75,p90,p95,p99) generate custom metrics / timeseries differently than gauges, counts, histograms and distributions with nonpercentile aggregations (sum, count, min, max, avg). Because percentiles aren't reaggregatable, Datadog preserves 5 timeseries for every potentially queryable tag combinations. This is different from the # of custom metrics generated from gauges, counts, histograms or distributions with nonpercentile aggregations (dependent on the unique # of tag value combinations that appear in your data).
```

Comment:
```suggestion
Distribution metrics with percentile aggregations (`p50`, `p75`, `p90`, `p95`, `p99`) generate custom metrics or timeseries differently than gauges, counts, histograms, and distributions with nonpercentile aggregations (`sum`, `count`, `min`, `max`, `avg`). Because percentiles aren't reaggregatable, Datadog preserves five timeseries for every potentially queryable tag combination. This is different from the number of custom metrics generated from gauges, counts, histograms, or distributions with nonpercentile aggregations (dependent on the unique number of tag value combinations that appear in your data).
```